doc/k8s/README.md

```diff
@@ -0,0 +1,53 @@
+# Running SQLFlow on Kubernetes
+
+This is a tutorial on how to run SQLFlow on Kubernetes, this tutorial will deploy:
+- MySQL server with the SQLFlow example datasets loaded, and
+- SQLFlow component contains the sqlflow-server the Jupyter notebook.
+
+Then you can run the SQLFlow query in the Jupyter notebook on the browser.
+
+## Preparation
+
+1. Kubernetes cluster: You can check the [offical page](https://kubernetes.io/docs/setup/) to set up a
+Kubernetes cluster. This tutorial would use [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+which can the Kubernetes locally to demonstrate the SQLFlow.
+1. [Install the kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), which is the command line tool
+to interact with the Kubernetes cluster.
+1. Build the SQLFlow Docker image or using the official Docker image directly: [sqlflow/sqlflow:latest](https://hub.docker.com/r/sqlflow/sqlflow).
```

Comment:
pull the official Docker image or build ... should be better? Since we don't quite likely want to bother users to build images.

Reply:
done.